pkg/consensus/consensus.go

```diff
@@ -292,6 +303,27 @@ events:
 			s.log.Debug("timer fired",
 				zap.Uint32("height", hv.Height),
 				zap.Uint("view", uint(hv.View)))
+			if s.Config.ProtocolConfiguration.EnableDBFTWatchdog {
```

Comment:
I think we need to use another timer which starts with new block approval. It should depend on `SecondsPerBlock`, last block timestamp and the current time.

Reply:
Fixed. But this way of consensus restart is also imperfect, because timer can't guarantee simultaneous node restart:
```
Timer that will send the current time on its channel after at least duration d.
```